mw_jwt.go

```diff
@@ -206,22 +225,41 @@ func (k *JWTMiddleware) getBasePolicyID(token *jwt.Token) (string, bool) {
 	return "", false
 }
 
+func (k *JWTMiddleware) getUserIdFromClaim(token *jwt.Token) (string, error) {
+	var userId string
+	var found bool
+
+	if k.Spec.JWTIdentityBaseField != "" {
+		userId, found = token.Claims.(jwt.MapClaims)[k.Spec.JWTIdentityBaseField].(string)
+
+		if !found {
+			log.Warning("Base Field not found, falling back to SUB")
+		}
+	}
+	if !found {
+		userId, found = token.Claims.(jwt.MapClaims)[SUB].(string)
+		if !found {
+			message := fmt.Sprint("user id was not found in claims")
+			log.Error(message)
+			return "", errors.New(message)
+		}
+	}
+	log.Debugf("Found User Id: ", userId)
```

Comment:
This looks like a bug: shouldn't it be something like: `log.Debugf("Found User Id: %s", userId)`

Reply:
good point, either this or replica function to just `log.Debug`

Reply:
of course. will check what had happened